doc/v2-manifesto.md

```diff
@@ -0,0 +1,27 @@
+## What sucks about Squirrel v1's Design™
+
+* Squirrel.Windows and Squirrel.Mac are completely different
+  - This confuses the living fuck out of people
+
+* Squirrel.Mac is missing lots of features
+  - Delta updates
+  - Updates via static HTTPS (i.e. S3)
+  - Rollout updates 
+
```

Comment:
Squirrel.Mac also doesn't launch the app with flags indicating an update was just installed, etc. Update "hooks" are only available on Squirrel.Windows. 

I'm not sure how often people utilize these hooks (at least in the Electron community) and on Mac the idea of launching a desktop app with flags and expecting it to quit would be pretty unusual. (Plus, only a single instance of an app can run at a time so it might be messy to implement.) Personally I'd advocate for dropping this feature entirely and letting people implement their own "hey look, I put 1.0.0 in localstorage and now it's 1.0.1 and I need to add my desktop shortcut" `did-just-update` detection.